src/universal/redux/makeReducer.js

```diff
@@ -6,15 +6,28 @@ import authToken from './authDuck';
 import {reducer as storageReducer} from 'redux-storage-whitelist-fn';
 import storageMerger from 'universal/redux/storageMerger';
 
-const currentReducers = {
+const initialReducers = {
   authToken,
   cashay: cashayReducer,
   form: formReducer,
   notifications
 };
+const currentReducers = {...initialReducers};
 
 export default (newReducers) => {
+  Object.assign(initialReducers, newReducers);
```

Comment:
shouldn't that be: `Object.assign(currentReducers, newReducers);`?
If we mutate the initialReducers, we'll only be able to do this trick once